src/server/engine_shard.cc

```diff
@@ -810,6 +810,12 @@ auto EngineShard::AnalyzeTxQueue() const -> TxQueueInfo {
 
   auto& db_slice = namespaces.GetDefaultNamespace().GetCurrentDbSlice();
 
+  {
+    auto value = queue->At(cur);
+    Transaction* trx = std::get<Transaction*>(value);
+    info.head.debug_id_info = trx->DebugId();
```

Comment:
tx_id + name of the command + shard count. Maybe we should also include per shard data, arguments etc ?